MagisterkaApp.Calculator/ReadFile.cs

```diff
@@ -89,10 +92,18 @@ public static ObservableCollection<FrequencyStep> ReadMonitoringFile(List<string
                             resultLine = filtredLine.Split(new char[] { ' ' }).ToArray();
                             resultLine = resultLine.Where(element => !string.IsNullOrWhiteSpace(element)).ToArray();
 
-                            var frequency = double.Parse(resultLine[frequencePosition]);
-                            var primary = double.Parse(resultLine[primaryPosition]);
-                            var secondaryOne = double.Parse(resultLine[secondaryOnePosition]);
-                            var secondarySecond = double.Parse(resultLine[secondaryTwoPosition]);
+                            var separator = CultureInfo.CurrentCulture.NumberFormat.NumberDecimalSeparator;
+
+
+                            var frequency = double.Parse(Regex.Replace(resultLine[frequencePosition], "[.,]", separator));
+                            var primary = double.Parse(Regex.Replace(resultLine[primaryPosition], "[.,]", separator));
```

Comment:
We don't need to change the reading of numbers from point to comma for Windows 10. This change is only for Windows 7.